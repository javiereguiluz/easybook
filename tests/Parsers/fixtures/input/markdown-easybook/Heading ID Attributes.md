setext-style headers (processed first by the parser)

(1) Headers with auto-ID:

Header 1
========

Header 2
--------

(2) Headers with custom ID attributes:

Header 1 {#my-custom-id-attribute}
========

Header 2 {#my-another-custom-id-attribute}
--------

atx-style headers (processed after the setext headers)

(1) Headers with auto-ID:

# Header 1

## Header 2

### Header 3

#### Header 4

##### Header 5

###### Header 6

(2) Repeat the same headings to ensure that the ID are unique:

# Header 1

## Header 2

### Header 3

#### Header 4

##### Header 5

###### Header 6

(3) Repeat again the same headings, but with the complete atx-style syntax:

# Header 1 #

## Header 2 ##

### Header 3 ###

#### Header 4 ####

##### Header 5 #####

###### Header 6 ######

(4) Headers with custom ID attributes:

# Header 1 {#my-custom-id-1}

## Header 2 {#my-custom-id-2}

### Header 3 {#my-custom-id-3}

#### Header 4 {#my-custom-id-4}

##### Header 5 {#my-custom-id-5}

###### Header 6 {#my-custom-id-6}

(5) Headers with custom ID attributes and using the complete atx-style syntax:

# Header 1 # {#my-other-custom-id-1}

## Header 2 ## {#my-other-custom-id-2}

### Header 3 ### {#my-other-custom-id-3}

#### Header 4 #### {#my-other-custom-id-4}

##### Header 5 ##### {#my-other-custom-id-5}

###### Header 6 ###### {#my-other-custom-id-6}
