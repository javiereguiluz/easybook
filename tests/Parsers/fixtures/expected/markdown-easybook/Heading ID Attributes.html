<p>setext-style headers (processed first by the parser)</p>

<p>(1) Headers with auto-ID:</p>

<h1 id="header-1">Header 1</h1>

<h2 id="header-2">Header 2</h2>

<p>(2) Headers with custom ID attributes:</p>

<h1 id="my-custom-id-attribute">Header 1</h1>

<h2 id="my-another-custom-id-attribute">Header 2</h2>

<p>atx-style headers (processed after the setext headers)</p>

<p>(1) Headers with auto-ID:</p>

<h1 id="header-1-2">Header 1</h1>

<h2 id="header-2-2">Header 2</h2>

<h3 id="header-3">Header 3</h3>

<h4 id="header-4">Header 4</h4>

<h5 id="header-5">Header 5</h5>

<h6 id="header-6">Header 6</h6>

<p>(2) Repeat the same headings to ensure that the ID are unique:</p>

<h1 id="header-1-3">Header 1</h1>

<h2 id="header-2-3">Header 2</h2>

<h3 id="header-3-2">Header 3</h3>

<h4 id="header-4-2">Header 4</h4>

<h5 id="header-5-2">Header 5</h5>

<h6 id="header-6-2">Header 6</h6>

<p>(3) Repeat again the same headings, but with the complete atx-style syntax:</p>

<h1 id="header-1-4">Header 1</h1>

<h2 id="header-2-4">Header 2</h2>

<h3 id="header-3-3">Header 3</h3>

<h4 id="header-4-3">Header 4</h4>

<h5 id="header-5-3">Header 5</h5>

<h6 id="header-6-3">Header 6</h6>

<p>(4) Headers with custom ID attributes:</p>

<h1 id="my-custom-id-1">Header 1</h1>

<h2 id="my-custom-id-2">Header 2</h2>

<h3 id="my-custom-id-3">Header 3</h3>

<h4 id="my-custom-id-4">Header 4</h4>

<h5 id="my-custom-id-5">Header 5</h5>

<h6 id="my-custom-id-6">Header 6</h6>

<p>(5) Headers with custom ID attributes and using the complete atx-style syntax:</p>

<h1 id="my-other-custom-id-1">Header 1</h1>

<h2 id="my-other-custom-id-2">Header 2</h2>

<h3 id="my-other-custom-id-3">Header 3</h3>

<h4 id="my-other-custom-id-4">Header 4</h4>

<h5 id="my-other-custom-id-5">Header 5</h5>

<h6 id="my-other-custom-id-6">Header 6</h6>
